src/java.management/share/classes/java/lang/management/ThreadMXBean.java

```diff
@@ -526,10 +526,11 @@ public interface ThreadMXBean extends PlatformManagedObject {
 
     /**
      * Tests if the Java virtual machine implementation supports CPU time
-     * measurement for any thread.
+     * measurement for any platform thread.
```

Comment:
Yes, this make sense although getting the word right for isCurrentThreadCpuTimeSupported is tricky because its about the caller of getCurrentThreadCpuTime/getCurrentThreadUserTime rather than the caller of isCurrentThreadCpuTimeSupported. Hopefully it is clearer now.